src/schema/resolvers/addUser.test.js

```diff
@@ -0,0 +1,36 @@
+/* global jest, it, expect, describe */
+
+import addUserResolver from './addUser'
+import models from '../../models'
+
+jest.mock('../../models')
+
+it('should export an object by default', () => {
+  expect(addUserResolver).toBeInstanceOf(Object)
+})
+
+describe('default export', () => {
+  it('should have a Mutation property of Object type', () => {
+    expect(addUserResolver).toHaveProperty('Mutation', expect.any(Object))
+  })
+
+  describe('Mutation property', () => {
+    it('should have an addUser function', () => {
+      expect(addUserResolver.Mutation).toHaveProperty('addUser', expect.any(Function))
+    })
+
+    describe('addUser ', function () {
```

Comment:
What do you think about splitting up these tests?
```
describe('addUser', ...
// assertions on the resolver itself (exists, is a function, etc)
  describe('calling it', ...
  // assertions on its behavior
```

That way you can make use of Jest hooks and have smaller tests.